### DISCUSSION TITLE
Compression ratio in tif_to_jp2()

### DISCUSSION THREAD

**Discussion by ravishbapna** [opengeos/leafmap#428] [Q&A] [answered]
Please help me with the default compression ratio while converting to JP2 format using tif_to_jp2(). Also, please share how I can change the compression ratio while preparing JP2 file from GeoTIFF file.

**Comment by giswqs** [comment, marked as answer]
See https://github.com/opengeos/leafmap/pull/430. e.g., `creationOptions=["QUALITY=20"]`
